docs/odoo/usage/install-module.rst

```diff
@@ -7,12 +7,35 @@
 
 .. warning:: ``Import Module`` tool (import from a zip  file) doesn't work for modules with python files. It means that it doesn't work in most cases
 
+.. image:: ../../images/import-11.png
+
+.. image:: ../../images/import-11-1.png
+
 From zip archive
 ================
 
 * unzip module to your addons folder
 * restart odoo server
 
+11.0+
+-----
+
+install
+~~~~~~~
+
+* :doc:`activate developer mode <debug-mode>`
+* navigate to ``Apps`` menu
+* click ``Update Apps List`` 
+* search and open a module you need
+* click ``[Install]``
+
+update
+~~~~~~
+
+* :doc:`activate developer mode <debug-mode>`
```

Comment:
No need to activate developer mode to update module. Fix it in other sections too